coldfront/core/portal/views.py

```diff
@@ -32,6 +34,9 @@ def home(request):
 
         abc_projects, savio_projects, vector_projects = set(), set(), set()
         for project in project_list:
+            project.display_status = AllocationUserAttribute.objects.get(allocation__project=project,
```

Comment:
An error arises for users that do not have this attribute yet--try logging in as this user, for example:
```
from django.contrib.auth.models import User
User.objects.filter(username__endswith="edu").first()
```

A better approach might be to retrieve all 'Cluster Account Status' attributes for this user outside of the `for` loop, and build a hash table from the project name to the value. Set the `display_status` to the value if the project is in the table; otherwise set it to some null value you'll check for in the template.

Reply:
gotcha!